O/OpenCV/build_tarballs.jl

```diff
@@ -123,8 +123,8 @@ dependencies = [
     Dependency(PackageSpec(name="Qt5Base_jll", uuid="ea2cea3b-5b76-57ae-a6ef-0a8af62496e1"))
     Dependency(PackageSpec(name="Libglvnd_jll", uuid="7e76a0d4-f3c7-5321-8279-8d96eeed0f29"))
     BuildDependency(PackageSpec(name="libjulia_jll"))
-    Dependency(PackageSpec(name="libcxxwrap_julia_jll"))
+    Dependency(PackageSpec(name="libcxxwrap_julia_jll", uuid="3eaa8342-bff7-56a5-9981-c04077f7cee7"); compat="0.11")
```

Comment:
Why not libcxxwrap_julia_jll@0.12?

Reply:
Just to say, we are also still at 0.11 for some of our packages because we'd need to coordinate a bunch of packages to simultaneously make the jump to the newer version. Not sure if that's the case here, but it is a problem (and of course gets worse for people who'd like to combine packages, some of which made the jump already, some didn't).

Reply:
@fingolfin apologies for the obtuseness, but what are the 0.11 and 0.12 numbers?  Do they refer to Julia, Yggdrasil or BinaryBuilder?

Reply:
> Why not libcxxwrap_julia_jll@0.12?

Good point.

Reply:
The 0.11 and 0.12 are the `libcxxwrap_julia` version, there is a breaking change between 0.11 and 0.12 and you need at least 0.11 to be compatible with Julia 1.10. I would recommend to jump to 0.12 if you can, but it may require some changes to the OpenCV julia wrapper code.

Reply:
What's the latest on this now? Should this go to 0.13?

Reply:
Yes, 0.13.2 is the one used by the latest released CxxWrap.